### PR TITLE
create company card by schema provided

### PR DESCRIPTION
**Pull request by lucaslyl**
linear: https://linear.app/cardstack/issue/CS-7414/create-company-card

1. Per the previous discussion, we agreed to create one pr for the stand alone company card for its reusable functionality for the atom view.

Result of atom view:
image

### CODE REVIEW

File: packages/experiments-realm/crm/company.gts
Comment (lucaslyl) [inline packages/experiments-realm/crm/company.gts:61]: @tintinthong , I'll hold off on implementing the phone field until your phoneField PR is approved. Once it's merged, I'll proceed with the implementation.

File: packages/experiments-realm/Company/9203c173-3420-44b3-863d-90defd7c57c8.json
Comment (tintinthong) [inline packages/experiments-realm/Company/9203c173-3420-44b3-863d-90defd7c57c8.json]: Why does this start with capital letter?

**Review by tintinthong — APPROVED** [review]

File: packages/experiments-realm/Company/9203c173-3420-44b3-863d-90defd7c57c8.json
Comment (lucaslyl) [reply packages/experiments-realm/Company/9203c173-3420-44b3-863d-90defd7c57c8.json]: You're right. In my second commit, I changed PhoneNumberField to phone, but I forgot to update the object property in the existing card instances. I’ll make sure to fix that.

**Pull request merged**
